internal/controller/replication.storage/status.go

```diff
@@ -26,18 +26,21 @@ import (
 // sets conditions when volume was promoted successfully.
 func setPromotedCondition(conditions *[]metav1.Condition, observedGeneration int64) {
 	setStatusCondition(conditions, &metav1.Condition{
+		Message:            "Volume is promoted",
 		Type:               v1alpha1.ConditionCompleted,
 		Reason:             v1alpha1.Promoted,
 		ObservedGeneration: observedGeneration,
 		Status:             metav1.ConditionTrue,
 	})
 	setStatusCondition(conditions, &metav1.Condition{
+		Message:            "Volume is healthy",
```

Comment:
This message does not add enough info toe understand the meaning of the condition.

Reply:
since this is a sucessful condition, we dont have any information to be printed, VR is in healthy state. Can you please share some examples of what other information you want to get printed here?